pkg/proxy/iptables/proxier.go

```diff
@@ -1285,17 +1274,13 @@ func (proxier *Proxier) syncProxyRules() {
 		glog.Errorf("Failed to execute iptables-restore: %v", err)
 		// Revert new local ports.
 		glog.V(2).Infof("Closing local ports after iptables-restore failure")
```

Comment:
no need for this at all - at worst a comment explaining that newLocalPorts is transactional and will be abandoned.